.changeset/shaggy-islands-mix.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-kubernetes': major
+'@backstage/plugin-kubernetes-backend': major
+---
+
+Restructure configuration; Add GKE cluster locator
```

Comment:
Not required, but always good if you can give a few pointers how to migrate at least at a high level